src/Main.java

```diff
@@ -1,9 +1,37 @@
+import java.util.Scanner;
 
 public class Main {
     public static void main(String[] args) {
 
-        // Hier mag je je code schrijven voor de hoofd-opdracht
+        Integer[] numeric = {1, 2, 3, 4, 5, 6, 7, 8, 9, 0};
+        String[] alphabetic = {"een", "twee", "drie", "vier", "vijf", "zes", "zeven", "acht", "negen", "nul"};
 
+        Translator t1 = new Translator(numeric, alphabetic);
+        // System.out.println(t1.translate(9)); // Testen of de methode werkt: ja, returnt "negen" bij meegegeven van 9.
 
+        boolean play = true; // translator is actief als play true is
+        String invalid = "ongeldige invoer";
+        Scanner scanner = new Scanner(System.in);
+
+        while (play) {
+            System.out.println("Type 'x' om te stoppen \nType 'v' om te vertalen");
+            String input = scanner.nextLine(); // Aanspreken van de scanner
+
+            if (input.equals("x")) {
```

Comment:
.equals doet de truuk, ik zou zelf echter adviseren om .equalsIgnoreCase te gebruiken. Stel dat iemand een hoofdletter X gebruikt, dat werkt het meteen niet meer. Met .equalsIgnoreCase kan je dat voorkomen!

Reply:
Dat is een goeie, ik pas het aan!